scopes/component/component-compare/diff-cmd.ts

```diff
@@ -6,16 +6,17 @@ import { ComponentCompareMain } from './component-compare.main.runtime';
 export class DiffCmd implements Command {
   name = 'diff [values...]';
   group = 'development';
-  description = "show the diff between the components' source files and config";
+  description =
+    "show the diff between the components' current source files and config, and their latest snapshot or tag";
```

Comment:
i dont think so - it's the current source files and config (no need for a , as there are are only two items), and then diff between that and their (the component's) latest snapshot

It might not be clear as it is, so we can reword, but your wording isnt what it should say.